modules/perception/camera/tools/offline/visualizer.cc

```diff
@@ -378,8 +378,9 @@ double Visualizer::regularize_angle(const double radian_angle) {
     return radian_angle + M_PI * 2.0;
   } else if (radian_angle > M_PI) {
     return radian_angle - M_PI * 2.0;
+  } else {
+    return radian_angle;
```

Comment:
No need to be in `else`?

Reply:
radian_angle is covered by comparing condition, therefore, use else is properly and that also refine the code style IMO.